### ISSUE TITLE
importNetFromTensorflow, dropout switch error

### ISSUE THREAD

**Issue by andyl1992** [opencv/opencv_contrib#1454]
##### System information (version)
- OpenCV => 3.3
- Operating System / Platform => Windows 64 Bit
- Compiler => Visual Studio 2015

##### Detailed description
I have created and trained a cnn model using keras, then using tensor-flow saved the trained model as a .pb file.

The issue is that when i import this model in openCV, i get error unknown layer type switch in op
dropout_1

**Comment by alalek** [comment]
Try latest master (DNN module is moved into main opencv/opencv repository): https://github.com/opencv/opencv/pull/9673
Closing, because we don't fix previous releases from the master branch (all fixes are pushed on development branch only).

**Comment by andyl1992** [comment]
i changed my branch to 3.3.1-dev this morning and the problem still persists.

Is this the duplicate issue your referring to? 
https://github.com/opencv/opencv/pull/9673
